includes/migration-guide/runtime/asp/ipad-should-not-be-used-custom-capabilities-file-because-it-now-browser.md

```diff
@@ -13,3 +13,5 @@ If iPad-specific capabilities are required, it is necessary to modify iPad behav
 | Scope   |Edge|
 |Version|4.5|
 |Type|Runtime|
+
+<!-- TODO: Should this include #### Affected APIs heading with text "None" under it? -->
```

Comment:
cc: @gewarren

I also met an include that seem to indicate affected APIs, but indicates that in the Details or Suggestion heading.

Reply:
The text should say "Not detectable via API analysis".